app/views/acm/index.haml

```diff
@@ -53,17 +53,6 @@
   * [View conference archives](#{url_for('acm/conference/about')}#archive)
   * [Get details about the next Link-State conference](#{url_for('acm/conference')})
 
-%h2#hackathon
```

Comment:
Is it worth leaving a link as a "related event"?

Reply:
Perhaps. I removed it since  hackcwru is it's own seperate organization now.